infra/bisector.py

```diff
@@ -163,5 +151,34 @@ def bisect(old_commit, new_commit, test_case_path, fuzz_target, build_data):  #
     return commit_list[new_idx]
 
 
+def bisect(old_commit, new_commit, test_case_path, fuzz_target, build_data):  # pylint: disable=too-many-locals
+  """From a commit range, this function caluclates which introduced a
+  specific error from a fuzz test_case_path.
+
+  Args:
+    old_commit: The oldest commit in the error regression range.
+    new_commit: The newest commit in the error regression range.
+    test_case_path: The file path of the test case that triggers the error
+    fuzz_target: The name of the fuzzer to be tested.
+    build_data: a class holding all of the input parameters for bisection.
+
+  Returns:
+    The commit SHA that introduced the error or None.
+
+  Raises:
+    ValueError: when a repo url can't be determine from the project.
+  """
+  result = _bisect(old_commit, new_commit, test_case_path, fuzz_target,
+                   build_data)
+
+  # Clean up projects/ as _bisect may have modified it.
+  oss_fuzz_repo_manager = repo_manager.BaseRepoManager(helper.OSS_FUZZ_DIR)
+  oss_fuzz_repo_manager.git(['reset', 'projects'])
```

Comment:
nit: I'm wondering if `projects` is even going to have dynamic value, maybe we should pass it as an argument or move into a constant

Reply:
hmm, I don't think this will change. If it does, I can make this an argument.